static/js/click_handlers.js

```diff
@@ -189,6 +189,40 @@ exports.initialize = function () {
         var local_id = $(this).attr('data-reaction-id');
         var message_id = rows.get_message_id(this);
         reactions.process_reaction_click(message_id, local_id);
+        $(".tooltip").remove();
+    });
+
+    // TOOLTIP FOR MESSAGE REACTIONS
+
+    $('#main_div').on('mouseenter', '.message_reaction', function (e) {
+        e.stopPropagation();
+        var elem = $(this);
+        var local_id = elem.attr('data-reaction-id');
+        var message_id = rows.get_message_id(this);
+        var title = reactions.get_reaction_title_data(message_id, local_id);
+
+        elem.tooltip({
+            title: title,
+            trigger: 'hover',
+            placement: 'bottom',
+            animation: false,
+        });
+        elem.tooltip('show');
+        $(".tooltip, .tooltip-inner").css('max-width', "600px");
```

Comment:
Can we set this width in CSS files (by e.g. adding a class to the tooltip here)?

How does this work in narrow screens and mobile web -- do we need conditionals for that case?

Reply:
For mobile browsers and narrow screens in general, the tooltip changes size to fit the browser size. This behavior comes with bootstrap.

Reply:
I tried adding a custom template for the tooltip and then setting the width in the CSS files but that didn't seem to work. I have left this as it for now.

As far as the resizing for smaller screens goes, here's a screenshot for what it looks like in mobile browser view:

![mobile browser reaction](https://user-images.githubusercontent.com/25329595/61528377-34aca480-aa3c-11e9-8e4b-984111d30312.png)